### ISSUE TITLE
Travis build failing on stetho-js-rhino

### ISSUE THREAD

**Issue by jasta** [facebook/stetho#354]
Build snippet:
```
:stetho-js-rhino:transformClassesWithDexForDebugAndroidTest FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':stetho-js-rhino:transformClassesWithDexForDebugAndroidTest'.
> com.android.ide.common.process.ProcessException: org.gradle.process.internal.ExecException: Process 'command '/usr/lib/jvm/java-7-oracle/bin/java'' finished with non-zero exit value 137

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED
```
Raw log: https://s3.amazonaws.com/archive.travis-ci.org/jobs/107107204/log.txt

**Comment by jasta** [comment]
This is believed to be caused by the Linux kernel killing the process for using too much memory.  The artifact being dexed here is enormous.

**Comment by jasta** [comment]
This issue is so frustrating.  Can't seem to fix it, I think we have to disable stetho-js-rhino CI :(

**Comment by jasta** [comment]
Fixed in #389
